README.md

```diff
@@ -138,6 +138,33 @@ Then try running `npm install` again.
 
 Please see [build instructions](https://github.com/deltachat/deltachat-core#building-your-own-libdeltachatso) for additional information.
 
+### Logging
+
+#### Logging Options
+[?]
+
+
+#### Logfile
+
+The Logfiles can are found in:
+```
+Linux: ~/.config/DeltaChat/logs/
+Mac: "~/Library/Application Support/DeltaChat/logs" [?]
+Windows: [?]
```

Comment:
should we remove windows, because we don't have it yet?

Reply:
We don't have it in the install instructions, so i would leave it out there too.

Reply:
Yes lets leave windows out until we have it. Don't know how long it still will take, could be some weeks.